examples/groth16-verifier/README.md

```diff
@@ -0,0 +1,29 @@
+# Groth16 Verifier Example
+
+This **experimental** example demonstrates the verification of a proof from another proof system, in this case Groth16 over the BN_254 curve (generated by Circom/SnarkJS), within the zkVM.
```

Comment:
```suggestion
This **experimental** example demonstrates the verification of a proof from another proof system, in this case Groth16 over the BN_254 curve, within the zkVM.
```
It's unclear in this sentence what "(generated by Circom/SnarkJS)" is applied to. At first I thought this was applied to BN_254, but that is not accurate. I'm guessing this is the Groth16 proof? Below this is more clear in the bullet list, so I'm suggesting this parenthetical be deleted.